pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -1271,6 +1271,13 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					PropertyName: "topic",
 					Required:     true,
 				},
+				{
+					Label:        "Add group key to topic",
+					Element:      ElementTypeCheckbox,
+					Description:  "If set, the group key will be appended to the topic.",
```

Comment:
It would be great if we explain what group key is. This term is only for implementation details and is not obvious to users.